### PR TITLE
impr(quotes): add 6 quotes (aikooo7)

### PR DESCRIPTION
**Pull request by aikooo7**
### Description

I have added six English quotes to the project.

My pull request includes six quotes that I found during a search, and they are in English.

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6331395328

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38688,6 +38688,42 @@
       "source": "Boisvert",
       "length": 670,
       "id": 6994
+    },
+    {
+      "text": "In the depth of winter, I finally learned that there was in me an invincible summer",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38693]: Missing period

Comment (Miodec) [reply frontend/static/quotes/english.json:38693]: Also probably change to `that there was an invincible summer in me`

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38688,6 +38688,42 @@
...
+      "text": "Do not be dismayed by the brokenness of the world. All things break. And all things can be mended. Not with time, as they say, but with intention. So go. Love intentionally, extravagantly, unconditionally. The broken world waits in darkness for the light that is you.",
+      "source": "L.R. Knos",
+      "length": 267,
+      "id": 6999
+    },
+    {
+      "text": "Don't watch the clock; do what it does. Keep going.",
+      "source": "Sam Levenson",
+      "length": 51,
+      "id": 7000
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38726]: Bit too short imo. Use a longer version if available or remove.

**Review by Miodec — CHANGES_REQUESTED** [review]
Some things to update

File: frontend/static/quotes/english.json (hunk 2 above)
Comment (aikooo7) [reply frontend/static/quotes/english.json]: Added bigger quote, let me you know if you want a bigger and if so size would help.

File: frontend/static/quotes/english.json (hunk 1 above)
Comment (aikooo7) [reply frontend/static/quotes/english.json:38693]: Done everything, appreciate your help.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6354966470

**Comment by aikooo7** [comment]
Just a note, my bad for all the quote size commits, sadly I always get it wrong, that ci for checking json is a really good feature.

**Review by Miodec — APPROVED** [review]

**Pull request merged**